Updated becoming-a-partner.md for DEVDOCS-1932 (#454)
bigcommerce/dev-docs: docs/api-docs/partner/becoming-a-partner.md

</div>
</div>

### Creating a Sandbox Store

1. Go to the BigCommerce Pricing page and Select [Start your free trial](https://www.bigcommerce.com/essentials/pricing/).
2. Where prompted for an email address, **enter the same email** that you used when applying to become a Partner.
3. In the trial store setup form, enter a store name, password, and other details. Then select **Create your store**.
4. Log in to your Partner account to access the [Partner Portal](https://partners.bigcommerce.com). 
5. Navigate to **Support > Contact Support** and select **Create a Support Case**.

> 6. Complete the form with **Partner Sandbox** as the subject line. In the case description, enter your new sandbox store’s canonical URL (for example: http://your-store-name.mybigcommerce.com) and request conversion to a sandbox. 

Reviewer: ```suggestion
6. While completing the form, use _Partner Sandbox_ as the subject line. In the case description, enter your new sandbox store’s canonical URL (for example: http://your-store-name.mybigcommerce.com) and request conversion to a sandbox. 

```

> 7. Under **Issue Type**, select **Merchant Account Services**. Click **Create Case**. We will flag this as a sandbox to ensure your continued access to the store, as opposed to the normal 15-day trial. This can take around two days.

Reviewer: ```suggestion
7. Under **Issue Type**, select **Merchant Account Services**. Click **Create Case**. We will flag this as a sandbox to ensure your store access continues beyond the 15-day trial period. This process can take up to 2 days.

```

<!-- theme: warning -->

### Trials Erased After 15 days
> Although a sandbox store is not required for app development, it is best practice. If you opt to develop on a trial store, keep in mind that the store and data will be erased after 15 days and a new trial store will have to be created.

</div>
</div>
</div>

### Creating a Sandbox Store

> 1. Go to the BigCommerce Pricing page and select [Start your free trial](https://www.bigcommerce.com/essentials/pricing/).

Reviewer: ```suggestion
1. Go to the [BigCommerce Pricing](https://www.bigcommerce.com/essentials/pricing/) page and select **Start your free trial**.

```

2. When prompted for an email address, enter the _same_ email you used when applying to become a partner.

3. In the trial store setup form, enter a store name, password, and other details. Then select **Create your store**.

4. Log into your Partner account to access the [Partner Portal](https://partners.bigcommerce.com). 

5. Navigate to **Support > Contact Support** and select **Create a Support Case**.

> 6. While completing the form, use _Partner Sandbox_ as the subject line. In the case description, enter your new sandbox store’s canonical URL (for example: http://your-store-name.mybigcommerce.com) and request conversion to a sandbox. 

Reviewer: ```suggestion
6. While completing the form, use "Partner Sandbox" as the subject line. In the case description, enter your new sandbox store’s canonical URL (for example: http://your-store-name.mybigcommerce.com) and request conversion to a sandbox. 

```

7. Under **Issue Type**, select **Merchant Account Services**. Click **Create Case**. We will flag this new trial store as a sandbox to ensure your store access continues beyond the 15-day trial period. This process can take up to 2 days.

> 8.  When you log into your new trial store, use the same email associated with your Partner login.

Reviewer: ```suggestion
8.  Log into your new trial store using the same email associated with your Partner login.

```

> While not all fields are required to publish edits for your listing, they are all highly recommended. Without filling in all fields, your listing will be missing information that merchants have seen on other listings and expect to see throughout the Marketplace.

Reviewer: ```suggestion
We highly recommend filling out all the fields associated with publishing edits for your listing. Without filling in all fields, your listing will lack information merchants are accustomed to seeing on other listings and expect to see throughout the Marketplace.

```

The changes will be effective immediately in your control panel app card, but the updates can take up to 24 hours to appear on the public BigCommerce Marketplace. Feel free to use this as a grace period to make additional edits as needed.

**Who should I contact if I have further questions?**

> Email <a href="mailto:appstore@bigcommerce.com">appstore@bigcommerce.com</a> if you have further questions.

Reviewer: ```suggestion
If you have additional questions, email <a href="mailto:appstore@bigcommerce.com">appstore@bigcommerce.com</a>.

```
